provisioner/buildinfo.go

```diff
@@ -1,4 +1,4 @@
-package main
+package provisioner
```

Comment:
The `BuildInfo` struct defines a ```SSHPublicKey string `json:"SSHPrivateKeyFile"```, the `json` looks wrong?

Reply:
Great catch, can you cover this as a follow-up PR, since this just moves files around